python/ray/experimental/workflow/common.py

```diff
@@ -29,6 +41,9 @@ def __init__(self, original_function: Callable,
         self._executed: bool = False
         self._output: Optional[WorkflowOutputType] = None
         self._step_id: StepID = uuid.uuid4().hex
```

Comment:
Can we prepend a human-readable name to the workflow id? This will make the storage format a bit easier to debug.